projects/Mastodon/default.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = ''
+      Mastodon is a free and open-source software platform for decentralized social networking.
+    '';
+    subgrants = [
+      "Mastodon"
+      "Mastodon-Quoting"
+    ];
+  };
+  nixos = {
+    modules.services.mastodon = {
+      module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/mastodon.nix";
+      examples.basic = null;
+    };
+    tests = {
+      standard = pkgs.mastodon.tests.mastodon.standard;
+      remote-databases = pkgs.mastodon.tests.mastodon.remote-databases;
```

Using tests from `nixosTests` causes examples to stop working for some reason. Let's just do this until we can investigate that:

```suggestion
      standard = "${sources.inputs.nixpkgs}/nixos/tests/web-apps/mastodon/standard.nix";
      remote-databases = "${sources.inputs.nixpkgs}/nixos/tests/web-apps/mastodon/remote-databases.nix";
```